### ISSUE TITLE
Errors when exporting to OpenFOAM

### ISSUE THREAD

**Issue by MoeEmara** [enGits/engrid#58]
I finished the "Complete Mesh for a damper in a duct" tutorial but I am getting some errors whenever I try to export the mesh to OpenFOAM (grid + solver parameters)

> This seems to be a bug in enGrid file: ..\..\..\libengrid\polymesh.cpp line: 1251
> This seems to be a bug in enGrid file: ..\..\..\libengrid\physicalboundarycondition.cpp line:176

All the solver parameters gets exported correctly but the constant/polyMesh folder is empty.
And when I try to export the grid only I get this error:

> The grid is not suitable for OpenFOAM export (e.g. missing volume mesh).
> file:  ..\..\..\libengrid\simplefoamwriter.cpp

**Comment by wyldckat** [comment]
A few questions:

  1. Which Operating System are you using?
  2. Which enGrid version are you using?
  3. Which installation instructions did you follow?

**Comment by MoeEmara** [comment]
1. Windows 10
2. dev-1.5 (btw this is the only version that I downloaded and didn't crash).
3. The recommended method for windows from here: https://github.com/enGits/engrid/wiki/%5BenGrid-1.4%5D-Installation#Recommended_Method_for_Windows

**Comment by wyldckat** [comment]
Sorry, but unfortunately that build of dev-1.5 is now considerably outdated and it was a snapshot of the source code that was in development at that time. Which means that it's only natural that it failed to run properly.

Nonetheless, I am intrigued as to why the version "enGrid 1.4.0 setup for Windows 64bit" that is on that page crashed at launch in your machine. Are you running enGrid in a real machine with Windows 10 or did you use it inside a virtual machine?

**Comment by MoeEmara** [comment]
Yes, I am running a real Windows 10 machine and both the 32 and 64-bit versions crashed at launch.
I tried to install it on Ubuntu 16.04 virtual machine but I got an error that `'/cae-team/ppa/ubuntu/dists/xenial/main/binary-amd64/Packages'` is not found and `engrid `package wasn't located!
